Example/wifi_sta/firmware/src/app.c

```diff
@@ -38,25 +39,63 @@
 // *****************************************************************************
 // *****************************************************************************
 
+#define ECC_TNG_DATA_ZONE                                       0x2
+#define ECC_TNG_GP_DATA_SLOT                                0x8
+#define ECC_TNG_GP_DATA_SLOT_BLOCK_BASE       0x0   
+
+static ATCA_STATUS status;    
+
 // *****************************************************************************
 /* Application Data
+ 
+ Summary:
+ Holds application data
+ 
+ Description:
+ This structure holds the application's data.
+ 
+ Remarks:
+ This structure should be initialized by the APP_Initialize function.
+ 
+ Application strings and buffers are be defined outside this structure.
+ */
 
-  Summary:
-    Holds application data
+APP_DATA appData;
+// *****************************************************************************
 
-  Description:
-    This structure holds the application's data.
+/* Work buffer used by littleFS file system during Format */
+uint8_t CACHE_ALIGN work[SYS_FS_LFS_MAX_SS];
 
-  Remarks:
-    This structure should be initialized by the APP_Initialize function.
+/* AP mode wifi-config struct used for FTP server*/
+SYS_WIFI_CONFIG sWifiCfg;
 
-    Application strings and buffers are be defined outside this structure.
-*/
+/*Flag to indicate use of ECC to store FTP server login credentials*/
+bool setECC = false;
 
-APP_DATA appData;
+// *****************************************************************************
+// *****************************************************************************
+// Section: Type Definitions
+// *****************************************************************************
+// *****************************************************************************
+/*struct used to store FTP server login credentials - username and password*/
+typedef struct client{
```

Comment:
Good to have structure definition in the header file

Reply:
done